.circleci/config.yml

```diff
@@ -106,6 +106,9 @@ jobs:
             LC_CTYPE: C
             LC_ALL: C
             LANG: C
+      - run:
```

Comment:
@bsipocz , are we moving this to a cron job in a separate PR?

Reply:
There is no circleCI cron, afaik, so no we need to rely on to have it run for the PRs. Tom was very optimistic that it won't fail (https://github.com/astropy/astropy/pull/8386#issuecomment-461344945) so we may give it a go this way (and if it's flaky then move the whole stuff to travis cron)

Reply:
Ah, I must have missed that conclusion. Thanks for the clarification!